loadJsonFile.py

```diff
@@ -0,0 +1,42 @@
+import googlemaps
+import json
+import requests
+
+API_KEY = 'AIzaSyDFtKtm0jnjckE-lB0LkO3SKlWmQIJush8';
+gmaps = googlemaps.Client(key=API_KEY)
+
+# 파일 읽기 -> result.json 로드하기
+with open('result.json') as f:
+	read_data = f.read()
+
+# dict type is 'list'
+dict = json.loads(read_data)
+
+# if the information is unknown, save 'unknown'
```

Comment:
If (first letter of the first word of the sentence should be capitalized)

Reply:
Same in other places

Reply:
OK! Thank you